### PR TITLE
Porting a Maui test MauiPrintUnitConvertTests to xUnit

### PR DESCRIPTION
**Pull request by John-Qiao**
Hi @RussKie and @AdamYoblick, could you please take a look at this pull and approve it if the changes are ok?

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/dotnet/winforms/pull/2405)

### CODE REVIEW

File: src/System.Windows.Forms/tests/IntegrationTests/MauiTests/MauiPrintUnitConvertTests/MauiPrintUnitConvertTests.cs [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+using ReflectTools;
+using ReflectTools.AutoPME;
+using WFCTestLib.Log;
+
+namespace System.Windows.Forms.IntegrationTests.MauiTests
+{
+    public class MauiPrintUnitConvertTests : XObject
+    {
+        private PrinterUnit _fromUnit;
+        private PrinterUnit _toUnit;
```
Comment (RussKie) [inline src/System.Windows.Forms/tests/IntegrationTests/MauiTests/MauiPrintUnitConvertTests/MauiPrintUnitConvertTests.cs]: I can't any reason for these to be fields, and thus shared across the type.
Please inline them at callsites.

File: src/System.Windows.Forms/tests/IntegrationTests/System.Windows.Forms.Maui.IntegrationTests/System.Windows.Forms.Maui.IntegrationTests.csproj [hunk 2]
```diff
@@ -5,7 +5,9 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\MauiTests\**\*.csproj" />
+    <ProjectReference Include="..\MauiTests\MauiButtonTests\MauiButtonTests.csproj" />
+    <ProjectReference Include="..\MauiTests\MauiComboBoxTests\MauiComboBoxTests.csproj" />
+    <ProjectReference Include="..\MauiTests\MauiPrintUnitConvertTests\MauiPrintUnitConvertTests.csproj" />
```
[the remark this answers is not in the document]
Comment (AdamYoblick) [reply src/System.Windows.Forms/tests/IntegrationTests/System.Windows.Forms.Maui.IntegrationTests/System.Windows.Forms.Maui.IntegrationTests.csproj]: This definitely should not change. The **\*.csproj was designed to auto include any maui tests that are added. Please revert the changes to this file.

**Comment by AdamYoblick** [comment]
I agree with Igor on this. We don't own these types, so there's no need for us to test them. And integration tests are not needed here since there is no UI interation, unit tests would be much better, as Igor said. But since corefx has these unit tests already, we don't need these anymore.

**Comment by John-Qiao** [comment]
Hi @RussKie and @AdamYoblick, thanks for your suggestions. As there's no UI interaction in this test case, so I closed this PR. 
Thanks again.

**Pull request closed without merging**